# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/27 15:37:39-05:00 haroldo.gamal@infolink.com.br 
#   [libata sata_sil] add another Seagate driver to blacklist
# 
# drivers/scsi/sata_sil.c
#   2005/01/27 15:37:33-05:00 haroldo.gamal@infolink.com.br +1 -0
#   [libata sata_sil] add another Seagate driver to blacklist
# 
diff -Nru a/drivers/scsi/sata_sil.c b/drivers/scsi/sata_sil.c
--- a/drivers/scsi/sata_sil.c	2005-01-30 06:59:30 -08:00
+++ b/drivers/scsi/sata_sil.c	2005-01-30 06:59:30 -08:00
@@ -90,6 +90,7 @@
 	{ "ST3120023AS",	SIL_QUIRK_MOD15WRITE },
 	{ "ST3160023AS",	SIL_QUIRK_MOD15WRITE },
 	{ "ST3120026AS",	SIL_QUIRK_MOD15WRITE },
+	{ "ST3200822AS",	SIL_QUIRK_MOD15WRITE },
 	{ "ST340014ASL",	SIL_QUIRK_MOD15WRITE },
 	{ "ST360014ASL",	SIL_QUIRK_MOD15WRITE },
 	{ "ST380011ASL",	SIL_QUIRK_MOD15WRITE },
